docs/onboarding/8_16/operator/onprem_kube_stack_values.yaml

```diff
@@ -0,0 +1,407 @@
+opentelemetry-operator:
+  manager:
+    extraArgs:
+      - --enable-go-instrumentation
+  admissionWebhooks:
+    certManager:
+      enabled: false
+
+autoGenerateCert:
+  enabled: true
+  recreate: true
+
+crds:
+  create: true
+
+defaultCRConfig:
+  image:
+    repository: "docker.elastic.co/beats/elastic-agent"
+    tag: "8.16.0-SNAPSHOT"
+  targetAllocator:
+    enabled: false
+  env:
+    - name: ELASTIC_AGENT_OTEL
+      value: '"true"'
+    - name: ELASTIC_ENDPOINT
+      valueFrom:
+        secretKeyRef:
+          name: elastic-secret-otel
+          key: elastic_endpoint
+    - name: ELASTIC_API_KEY
+      valueFrom:
+        secretKeyRef:
+          name: elastic-secret-otel
+          key: elastic_api_key
+    - name: K8S_NODE_NAME
+      valueFrom:
+        fieldRef:
+          fieldPath: spec.nodeName
+collectors:
+  cluster:
+    config:
+      exporters:
+        debug:
+          verbosity: basic
+        elasticsearch:
+          endpoints:
+          - ${env:ELASTIC_ENDPOINT}
+          api_key: ${env:ELASTIC_API_KEY}
+          mapping:
+            mode: ecs
```

Comment:
is it planned to be changed to `mode: otel`?
When we were discussing k8s part it was agreed to use `mode: otel` for the otel k8s integration

Reply:
Correct, we need to create a few pipelines with different ES mode. K8s should go with `mode: otel`, could you confirm that the `elasticinframetrics processor` won't be needed for the k8s receivers?

Reply:
> K8s should go with mode: otel

can you clarify? so for k8s metrics/logs I should add a different elasticsearch exporter configuration with `mode: otel`, correct?

> could you confirm that the elasticinframetrics processor won't be needed for the k8s receivers?

no, `elasticinframetrics` will be still needed, reason for that: avoid breaking the kibana inventory UI (for both k8s and system metrics views). Kubernetes overview dashboard itself will not relying on transformed metrics

Reply:
> can you clarify? so for k8s metrics/logs I should add a different elasticsearch exporter configuration with mode: otel, correct?

Exactly, I can push the needed changes.

> avoid breaking the kibana inventory UI

Will we have duplicated metrics then? Shouldn't we just push Otel native k8s metrics for this Onboarding?

Reply:
> Exactly, I can push the needed changes.

yes, it would be great, thanks!

> But then if I understood correctly, we will have "otel" and "ecs" metrics, have you tested if both of these metrics types can be sent with the ES exporter in otel mode? Will we need to conditionally route the metrics to different ES exporter (one with ecs mode and the oter in otel)?

very good point!
mostly I was testing configuration with the custom otel collector build (that includes elasticsearchexporter changes to support `otel` mode), I believe it was released in `v0.109.0` and should be already available in elastic-agent image, I will test it